xclim/core/indicator.py

```diff
@@ -1285,17 +1339,22 @@ def compute(*args, **kwds):
         """Compute the indicator.
 
         This would typically be a function from `xclim.indices`.
-        """
-        raise NotImplementedError
+        """  # numpydoc ignore=PR01
+        raise NotImplementedError()
 
-    def cfcheck(self, **das):
-        """Compare metadata attributes to CF-Convention standards.
+    def cfcheck(self, **das) -> None:
+        r"""Compare metadata attributes to CF-Convention standards.
 
         Default cfchecks use the specifications in `xclim.core.utils.VARIABLES`,
         assuming the indicator's inputs are using the CMIP6/xclim variable names correctly.
         Variables absent from these default specs are silently ignored.
 
         When subclassing this method, use functions decorated using `xclim.core.options.cfcheck`.
+
+        Parameters
+        ----------
+        \*\*das : dict
+            A dictionary of DataArrays to check.
```

Comment:
is this really a `dict`?  more like the arguments of a `dict`

Reply:
`das` is technically a dictionary because all arguments passed along are seen as members of a dictionary being populated in the call signature. We could be more specific about the types of the values accepted if we want (e.g. `dict of DataArray`).

The docstring needs _something_ and this is the closest generic type for the variable `das`.

Reply:
From the document you linked:

> When documenting variable length positional, or keyword arguments, leave the leading star(s) in front of the name and do not specify a type

Does that apply to \*\* ?

Reply:
Jumping in because I am confused too with the combination of both a r-string (`r""" .... """"`) _and_ the escaped stars `\*\*`. Shouldn't it be one of those, but no both ?

Reply:
I think this may have come about originally from the Indicator docstring parsing logic. This has now been addressed. No more `\` needed when the docstring is raw.